src/consensus/aft/raft.h

```diff
@@ -1022,6 +1032,13 @@ namespace aft
       const ccf::NodeId& to, Index start_idx, Index end_idx)
     {
       const auto prev_idx = start_idx - 1;
+
+      if (replica_state == Retired && start_idx >= end_idx)
+      {
+        channels->destroy_channel(to);
```

Comment:
Could we add a log line here to say that node `to` has been caught up after "our" retirement?

Reply:
Sure